src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -102,7 +103,13 @@ const Body = BaseForm.extend({
           }
         })
         .catch(() => {
-          Logger.error('Something unexpected happened during device probing.');
+          countProbedPorts++;
+          if (countProbedPorts === ports.length) {
+            Logger.error('No available ports. Loopback server failed and polling is cancelled.');
+            this.options.appState.trigger('invokeAction', 'authenticatorChallenge.cancel-polling');
+          } else {
+            Logger.error('Something unexpected happened during device probing.');
```

Comment:
Can you change the log message to something like `Authenticator is not listening on port ${currentPort}.`?